compass/ocean/tests/global_ocean/init/__init__.py

```diff
@@ -117,6 +121,14 @@ def validate(self):
             compare_variables(test_case=self, variables=variables,
                               filename1='initial_state/initial_state.nc')
 
+        if self.with_inactive_top_cells:
+            variables = [
+                'temperature', 'salinity', 'layerThickness', 'normalVelocity']
+            compare_variables(test_case=self, variables=variables,
+                              filename1='initial_state/initial_state_crop.nc',
+                              filename2='initial_state/initial_state_comp.nc',
+                              quiet=False, check_outputs=False)
+
```

Comment:
I'm guessing you manually copied `initial_state/initial_state_comp.nc` from the other test case.  I think this comparison would also work if you constructed the absolute path to the original output file from the other test case and also set `skip_if_step_not_run=False` (which is not ideal, and I'm working on a better approach as a separate PR).  To construct this path, you would need to use `self.inactive_top_comp_subdir` that I suggested above, and then do a bit of path manipulation: 
```suggestion
        if self.with_inactive_top_cells:
           # construct the work directory for the other test
            work_dir2 = os.path.join(self.base_work_dir, self.mpas_core, self.test_group, 
                                     self.inactive_top_comp_subdir)
            filename2 = os.path.join(work_dir2,  'initial_state/initial_state.nc)
            variables = [
                'temperature', 'salinity', 'layerThickness', 'normalVelocity']
            compare_variables(test_case=self, variables=variables,
                              filename1='initial_state/initial_state_crop.nc',
                              filename2=filename2, quiet=False, check_outputs=False, 
                              skip_if_step_not_run=False)

```